inject/src/main/java/io/micronaut/context/env/ConfigurationPath.java

```diff
@@ -120,6 +121,15 @@ static ConfigurationPath of(BeanDefinition<?>... definitions) {
      */
     int index();
 
+    /**
+     * @return the current property catalog
+     *
+     * @since 4.7.0
+     */
+    default PropertyCatalog propertyCatalog() {
```

Comment:
add nullability annotation to return type

Reply:
fixed